haas/cli.py

```diff
@@ -718,7 +718,7 @@ def show_node(node):
 
     q = check_clientlib_response(lambda: C.node.show(node))
     for item in q.items():
-        sys.stdout.write("%s\t  :  %s\n" % (item[0], item[1]))
+        print item
```

Comment:
Not sure why I changed i. Anyway,  changed it back to `sys.stdout.write` for consistency.